packages/dd-trace/test/plugins/promise.js

```diff
@@ -12,95 +13,129 @@ module.exports = (name, factory) => {
 
     describe(name, () => {
       withVersions(plugin, name, version => {
-        beforeEach(() => {
-          tracer = require('../..')
-        })
-
-        afterEach(() => {
-          return agent.close()
-        })
-
-        describe('without configuration', () => {
+        if (!versionRange || semver.intersects(version, versionRange)) {
```

Comment:
Why not just return instead of moving everything inside the if?

Reply:
indeed. updated